### ISSUE TITLE
Knn imputation

### ISSUE THREAD

**Issue by loeroverl** [Thorkroos/7703codes#1]
# knn imputation transform 
from numpy import isnan
import pandas as pd
from sklearn.impute import KNNImputer
# load dataset

dataframe = pd.read_csv("/Users/liumingkai/Downloads/archive/world-happiness-report copy.csv")

# define imputer
imputer = KNNImputer(n_neighbors=5, weights='uniform', metric='nan_euclidean')
# fit on the dataset
imputer.fit(dataframe)
# transform the dataset
Xtrans = imputer.transform(dataframe)
